### PR TITLE
🎉 New Source: Yotpo [Low code CDK]

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
*Developing new connector for the source: Yotpo*

Resolves:
[Yotpo](https://github.com/airbytehq/DeveloperWeek-Connector-Contest/issues/14)

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## Tests

<details open><summary><strong>Integration & Acceptance</strong></summary>
Full Test Results

image

</details>

## 🚨 User Impact 🚨
- No breaking changes, Just addition of new source
<details><summary><strong>New Connector: Source Yotpo</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [ ] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

</details>

### CODE REVIEW

**Comment by btkcodedev** [comment]
Please add hackathon-developer-week label

**Comment by btkcodedev** [comment]
Conflicts resolved :)
Thanks

**Comment by btkcodedev** [comment]
@marcosmarxm conflicts resolved from https://github.com/airbytehq/airbyte/pull/25145

**Comment by btkcodedev** [comment]
Fixed some expected records as the platform is auto rejecting some of the seeded content for some reason
Recent Test Results - All passed

image

File: airbyte-integrations/connectors/source-yotpo/source_yotpo/schemas/email_analytics.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "properties": {
+    "since": {
+      "type": ["null", "string"]
+    },
+    "until": {
+      "type": ["null", "string"]
+    },
+    "values": {
+      "type": ["null", "object"],
+      "properties": {}
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-yotpo/source_yotpo/schemas/email_analytics.json:15]: missing properties.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-yotpo/source_yotpo/schemas/email_analytics.json (hunk 1 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-yotpo/source_yotpo/schemas/email_analytics.json]: Resolved

**Review by marcosmarxm — APPROVED** [review]
Thanks for the contribution @btkcodedev

**Pull request merged**